src/common/turbo.ts

```diff
@@ -82,36 +129,41 @@ export type ArNSPurchaseReceipt = AoTokenCostParams & {
   createdDate: string;
 };
 
-export interface ArNSPaymentProvider {
+// Define separate provider interfaces
+export interface ArNSUnauthenticatedPaymentProvider {
   // TODO: have this return just the number, for generic payment providers
   /** Returns the cost of the action in the Payment Provider's native currency (winc for Turbo) */
   getPrice(params: AoTokenCostParams): Promise<number>;
   getArNSPriceDetails(params: AoTokenCostParams): Promise<{
     winc: string;
     mARIO: mARIOToken;
   }>;
+}
+
+export interface ArNSAuthenticatedPaymentProvider
+  extends ArNSUnauthenticatedPaymentProvider {
   initiateArNSPurchase(
     params: AoTokenCostParams & { processId?: TransactionId },
-    options: WriteOptions,
+    options: WriteOptions, // Note: WriteOptions might not be necessary here if unused
```

Comment:
remove it

Reply:
[336d52f](https://github.com/ar-io/ar-io-sdk/pull/458/commits/336d52f03cd5ed7cb49173e26f4e3838a9b7cf91)